src/fairchem/core/units/mlip_unit/mlip_unit.py

```
@@ -926,8 +940,13 @@ def __init__(
         )
 
         # TODO see placeholder comment in TrainEvalUnit as well
-        self.autocast_enabled = bf16
-        self.autocast_dtype = torch.bfloat16
+        self.autocast_enabled = bf16 or fp16
```

can we throw an error here if they are both set to True, ie: only one should turned on at a time